server/src/fights/fights.service.ts

```diff
@@ -10,6 +11,30 @@ export class FightsService {
     FightInterface
   >();
 
+  constructor() {
+    const fight: FightInterface = {
+      id: 'mockup',
+      state: FightState.Scheduled,
+  
+      mainJudgeId: 'main',
+      redJudgeId: 'red',
+      blueJudgeId: 'blue',
+  
+      mainJudgeSocket: null,
+      redJudgeSocket: null,
+      blueJudgeSocket: null,
+  
+      redPlayerId: 'player1',
+      bluePlayerId: 'player2',
+  
+      redEventsHistory: [],
+      blueEventsHistory: [],
+
+      timer: new Timer(1)
+    };
+    this.newFight(fight);
```

Comment:
Może zostać jako rozwiązanie tymczasowe, ale na przyszłość trzeba będzie pamiętać żeby to usunąć